packages/flutter_tools/test/test_test.dart

```diff
@@ -82,7 +82,7 @@ Future<Null> _testFile(String testName, int wantedExitCode, String workingDirect
       expect(haveSeenStdErrMarker, isFalse);
       haveSeenStdErrMarker = true;
     }
-    expect(outputLine, matches(expectationLine), verbose: true, reason: 'Full output:\n- - - -----8<----- - - -\n${output.join("\n")}\n- - - -----8<----- - - -');
+    expect(outputLine, matches(expectationLine), reason: 'Full output:\n- - - -----8<----- - - -\n${output.join("\n")}\n- - - -----8<----- - - -');
```

Comment:
> Since this was passing verbose:true, the removal of the verbose parameter will affect this test expectation.

Hmmm, actually, doesn't this just change the verbosity of the diagnostics in the event that the test fails?